templates/todo/task_list.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>TODO list</h1>
+    <button style="float: right; bottom: 50px; position: relative" class="btn btn-primary"
```

Comment:
Instead of using inline styles, consider defining a CSS class for your button and applying it consistently across your project. This will make your styling more maintainable and easier to update.